packages/light-client/src/index.ts

```diff
@@ -327,15 +327,11 @@ export class Lightclient {
         // Subscribe to head updates over SSE
         // TODO: Use polling for getLatestHeadUpdate() is SSE is unavailable
         this.api.events.eventstream(
-          [routes.events.EventType.lightclientOptimisticUpdate],
-          controller.signal,
-          this.onSSE
-        );
-        this.api.events.eventstream(
-          [routes.events.EventType.lightclientFinalizedUpdate],
+          [routes.events.EventType.lightClientOptimisticUpdate],
           controller.signal,
           this.onSSE
         );
+        this.api.events.eventstream([routes.events.EventType.lightClientFinalityUpdate], controller.signal, this.onSSE);
```

Comment:
could we do a single `eventstream()` api call?

Reply:
Yes, you must do a single eventstream call, doing 2 is very wasteful